README.md

```diff
@@ -7,3 +7,4 @@
 👤 **Author**
 
 - GitHub: [@fabriceMUKARAGE](https://github.com/fabriceMUKARAGE)

```

Comment:
- [ ] You have really done well but consider this: I think it would be a good idea to replace "Author" with your name so that the person that reads this readme knows the name of the owner of the project. I do suggest however that you use [the readme template provided by microverse](https://github.com/microverseinc/readme-template). It says so right here in the project requirements:

![image](https://user-images.githubusercontent.com/107795045/218872412-c101e31a-cf74-4c6b-89c5-e5e4a0bce437.png)

Reply:
@amon-cofie Thanks for the feedback